tasks/Sub-arctic survival.md

```diff
@@ -0,0 +1,35 @@
+# Summary
+> It is referred to as a team synergy exercise. It is an hypothetical scenario of a plane crash in a very cold area. The participants (around 6) have to individually rank 15 salvaged items that they think will be useful for the survival of the team, the next step is to debate with the team and cooperativelly make a new rank of the salvaged items.
+
+# References
+> Source: http://hushlphs.weebly.com/uploads/9/9/5/3/9953763/the_subartic_situation.pdf
+> Official website: https://subarcticsurvival.com
+> More detailed source: https://www.gvsu.edu/cms4/asset/903124DF-BD7F-3286-FE3330AA44F994DE/the_subarctice_survival_program_-_edited.pdf
+
+# Stimuli
+## The visual components
+> https://www.google.com/url?sa=i&url=http%3A%2F%2Fhushlphs.weebly.com%2Fuploads%2F9%2F9%2F5%2F3%2F9953763%2Fthe_subartic_situation.pdf&psig=AOvVaw1v56SSpGvmaWvD7ve12plA&ust=1612299842027000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCPC_0I3Lye4CFQAAAAAdAAAAABAD
+> Aditionally, the participants are informed more details about their situation, such as expectations of their rescue, their clothing, the terrain, and additional details about their belongings.
+
+## Materials for alternative versions of the experiment 
+> A variation of the "sub-arctic survival situation" was not found, whoever, the task itself is a variation of survival tasks, so it can be easily modified to have different survival conditions and still test team cooperation.
+
+# Procedure
+## Steps
+> The team members (no distiction of roles) receive general instructions regarding their conditions: The time of the day, locations, distance from safety, vegetation and terrain, clothing, health and belongings. Every team member has access to a topological map and detailed information about expected temperature variations, snowfall, sunrise and sunset times, and wind and temperature conditions that cause exposed flesh to freeze. The team must stay together.
+> Step 1: The members receive a list of items that they salvaged in the plane crash and must individually rank in the order of how important the item is to their survival.
+> Step 2: Without modifying the list in step 1, now the members must collectivelly make a new ranked list.
+
+
+## Rolls 
+> No Roles (really?)
+
+## Instructions
+> Apart from the previous steps, teams are given a time limit to finish their discussions.
+
+# Criteria
+## Performance calculation
+> The performance of the team is evaluated against the oficial ranking of sub-arctic survival experts. The closer the team's ranking is to the one of the experts, the better the synergy of the team. In the case of this survival task, it was observed that 90% of the time teams perform better than their average member and 50% of the times they perform better than their best member did individually.
+
```

Comment:
The correct answers in order are: 
1. waterproof matches, 2.hand axe, 3. heavy duty canvas, 4. sleeping bag, 5. gallon maple syrup,6. braided nylon rope, 7. snowshoes, 8. inner tube, 9. shaving kit and mirror, 10. flashlight, 11.bacardi rum, 12.alarm clock, 13.compass, 14.celestial navigation book, 15.water purification tablets